### ISSUE TITLE
Moving the Thumbnails (in gallery mode) to the left

### ISSUE THREAD

**Issue by ccasalicchio** [sachinchoolur/lightslider#289]
Is it possible to position the thumbnails to the left of the slide?
![screenshot_1](https://cloud.githubusercontent.com/assets/2466879/20432990/daff7636-ad88-11e6-9688-c58ea6a70567.png)

**Comment by xonecas** [comment]
I have a similar requirement, where I need to move the thumbstrip to the top. Is this at all possible?

**Comment by PixelZombie** [comment]
I hacked it into the plugin - don't have the time for a pull request.

in lightslider.js change line 396 to:
```javascript
$slide.parent().css('padding-left', (settings.vThumbWidth + settings.galleryMargin) + 'px');
```

and line 421 to:
```javascript
var gMargin = (settings.vertical) ? 'margin-right' : 'margin-top';
```

In addition you have to change or overwrite the following CSS-selector like this:


```css
.lSSlideOuter.vertical .lSGallery {
    right: auto;
    left: 0;
}
```
Hope this helps you guys ;-)

**Comment by ccasalicchio** [comment]
Thank you Frank, I'll test it out today!

​

**Comment by ccasalicchio** [comment]
Using your suggestion, I've tweaked a bit further, to avoid not using the
vertical to the right

​so, adding ​

​thumbsLeft: false | true

will render either to the left, or the default​

lightslider.js: line 237 - 243

if (settings.vertical) {
                    $slide.parent().addClass('vertical');
                    elSize = settings.verticalHeight;
                    $slide.css('height', elSize + 'px');
                    *if (settings.thumbsLeft) {
                       $slide.parent().addClass('thumbsLeft');
               }
}*

line 398 (after above change)

if (settings.vertical === true) {   var padding = settings.thumbsLeft
=== true ? 'padding-left': 'padding-right';
   $slide.parent().css(padding, (settings.vThumbWidth +
settings.galleryMargin) + 'px');}

 line 425 (after above change)

var gMargin = (settings.vertical) ? (settings.thumbsLeft) ?
'margin-right' : 'margin-left' : 'margin-top';

​lightslider.css: added after line 260

/* Thumbnails on the left */.lSSlideOuter.vertical.thumbsLeft .lSGallery {
    right: auto;
    left: 0;
}​

​[image: Inline image 1]​

​It seems to be working great now, thanks!​

**Comment by PixelZombie** [comment]
Yeah it's better that way, but I didn't need the config part on my project. Otherwise I would have done it with config and all ;-) 

Cheers!

**Comment by chelette** [comment]
fderdgdfd

**Comment by xu4ka** [comment]
This method no longer works.

**Comment by SilverFoxA** [comment]
Not sure if anyone find this helpful but based on my requirement here's what I did

```
.lSSlideOuter.vertical{
	padding-left: 55px;
}
.lSSlideOuter.vertical .lSGallery{
	left: 0;
}
```

You can make it dynamic by adding `jQuery` width function.

**Comment by Pankajbatham27** [comment]
1. Replace in ( lightslider.min.js ) 
(padding-right to padding-left)
(vertical?"margin-left" to vertical?"margin-right")

2. Add custom css

.lSSlideOuter.vertical .lSGallery{
    left: 0;
}

**Comment by webdiller** [comment]
> 1. Replace in ( lightslider.min.js )
>    (padding-right to padding-left)
>    (vertical?"margin-left" to vertical?"margin-right")
> 2. Add custom css
> 
> .lSSlideOuter.vertical .lSGallery{
> left: 0;
> }

Works good, thx :)
